src/main/java/eu/deltasource/internship/hotel/repository/GuestRepository.java

```diff
@@ -18,6 +18,13 @@
 
 	private final List<Guest> repository;
 
+
+	///////////////////
```

Comment:
If it's for test purposes, you can equivalently create a method in the test class, which creates a repository with a guest.